src/gui/systray.h

```diff
@@ -16,6 +16,7 @@
 #define SYSTRAY_H
 
 #include <QSystemTrayIcon>
+#include <QGuiApplication>
```

Comment:
is it not possible to just use a forward declaration when using it inside a `QPointer` instance ?

Reply:
it is, done